masterlist.yaml

```
@@ -20680,6 +20750,101 @@ plugins:
     after: [ 'CFTO.esp' ]
     tag: [ Factions ]
 
+  ### Patches from Additional Clockwork ***
+  # Clockwork Sorting Patches #
+  - name: 'ClockworkSorting_Food_CACO.esp'
+    clean:
+      - crc: 0x83840648
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
+  - name: 'ClockworkSorting_Ing_CACO.esp'
+    msg:
+      - <<: *alreadyInX
+        subs: ["\"Additional Clockwork - CACO & Rare Curios Ingredient Sorting Patch\" (ClockworkSorting_Ing_CCCurios_CACO.esp)"]
+        condition: 'active("ClockworkSorting_Ing_CCCurios_CACO.esp")'
+      - type: error
+        content: '"Additional Clockwork - Rare Curios Ingredient Sorting Patch (ClockworkSorting_Ing_CCCurios.esp)" is incompatible. Consider using the merged plugin that comes with the installer instead.'
+        condition: 'active("ClockworkSorting_Ing_CCCurios.esp") and not active("ClockworkSorting_Ing_CCCurios_CACO.esp")'
+    clean:
+      - crc: 0xfc69bf72
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
+  - name: 'ClockworkSorting_Ing_CCCurios.esp'
+    msg:
+      - <<: *alreadyInX
+        subs: ["\"Additional Clockwork - CACO & Rare Curios Ingredient Sorting Patch\" (ClockworkSorting_Ing_CCCurios_CACO.esp)"]
+        condition: 'active("ClockworkSorting_Ing_CCCurios_CACO.esp")'
+      - type: error
+        content: '"Additional Clockwork - CACO Ingredient Sorting Patch" (ClockworkSorting_Ing_CACO.esp) is incompatible. Consider using the merged plugin instead.'
+        condition: 'active("ClockworkSorting_Ing_CCCurios_CACO.esp") and not active("ClockworkSorting_Ing_CCCurios_CACO.esp")'
+    clean:
+      - crc: 0x30bafdf7
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
```

Formatting changes + fixes a condition + replaces unaliased message with `*useInstead` message:

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
    msg:
      - <<: *alreadyInX
        subs: [ '"Additional Clockwork - CACO & Rare Curios Ingredient Sorting Patch" (ClockworkSorting_Ing_CCCurios_CACO.esp)' ]
        condition: 'active("ClockworkSorting_Ing_CCCurios_CACO.esp")'
      - <<: *useInstead
        subs: [ '"Additional Clockwork - CACO & Rare Curios Ingredient Sorting Patch" (ClockworkSorting_Ing_CCCurios_CACO.esp)' ]
        condition: 'active("ClockworkSorting_Ing_CACO.esp") and not active("ClockworkSorting_Ing_CCCurios_CACO.esp")'
    clean:
      - crc: 0x30BAFDF7
        util: 'SSEEdit v4.0.3'
```